proposals/dao/pcv_guardian_v3.ts

```diff
@@ -0,0 +1,131 @@
+  const pcvGuardian = await pcvGuardianFactory.deploy(addresses.core, safeAddresses);
+  await pcvGuardian.deployed();
+  logging && console.log(`pcvGuardian: ${pcvGuardian.address}`);
+
+  const fuseWithdrawalGuardFactory = await ethers.getContractFactory('FuseWithdrawalGuard');
+  const fuseWithdrawalGuard = await fuseWithdrawalGuardFactory.deploy(
+    addresses.core,
+    [
+      addresses.rariPool8FeiPCVDeposit,
+      addresses.rariPool8DaiPCVDeposit,
+      addresses.rariPool8LusdPCVDeposit,
+      addresses.rariPool79FeiPCVDeposit,
+      addresses.rariPool128FeiPCVDeposit,
+      addresses.rariPool22FeiPCVDeposit,
+      addresses.rariPool24FeiPCVDeposit,
+      addresses.rariPool18FeiPCVDeposit,
+      addresses.rariPool6FeiPCVDeposit,
+      addresses.turboFusePCVDeposit
+    ],
+    [
+      addresses.daiFixedPricePSM,
+      addresses.daiFixedPricePSM,
+      addresses.lusdHoldingPCVDeposit,
+      addresses.daiFixedPricePSM,
+      addresses.daiFixedPricePSM,
+      addresses.daiFixedPricePSM,
+      addresses.daiFixedPricePSM,
+      addresses.daiFixedPricePSM,
+      addresses.daiFixedPricePSM,
+      addresses.daiFixedPricePSM
+    ],
+    [
+      addresses.fei,
+      addresses.dai,
+      addresses.lusd,
+      addresses.fei,
+      addresses.fei,
+      addresses.fei,
+      addresses.fei,
+      addresses.fei,
+      addresses.fei,
+      addresses.fei
+    ],
+    [e18.mul(191_000), e18.mul(64_000), e18.mul(4_000), e18.mul(100_000), 0, 0, 0, 0, 0, 0]
+  );
+  await fuseWithdrawalGuard.deployed();
+  logging && console.log(`fuseWithdrawalGuard: ${fuseWithdrawalGuard.address}`);
+
+  return {
+    pcvGuardian,
+    fuseWithdrawalGuard
+  };
+};
+
+// Do any setup necessary for running the test.
+// This could include setting up Hardhat to impersonate accounts,
+// ensuring contracts have a specific state, etc.
+const setup: SetupUpgradeFunc = async (addresses, oldContracts, contracts, logging) => {
+  // make sure ETH oracle is fresh (for B.AMM not to revert, etc)
+  // Read Chainlink ETHUSD price & override chainlink storage to make it a fresh value
+  ethPrice8Decimals = Math.round((await contracts.chainlinkEthUsdOracleWrapper.read())[0].toString() / 1e10).toString();
+  await overwriteChainlinkAggregator(addresses.chainlinkEthUsdOracle, ethPrice8Decimals, '8');
+
+  // read pcvStats before proposal execution
+  pcvStatsBefore = await contracts.collateralizationOracle.pcvStats();
+};
+
+// Tears down any changes made in setup() that need to be
+// cleaned up before doing any validation checks.
+const teardown: TeardownUpgradeFunc = async (addresses, oldContracts, contracts, logging) => {
+  console.log(`No actions to complete in teardown for fip${fipNumber}`);
+};
+
+// Run any validations required on the fip using mocha or console logging
+// IE check balances, check state of contracts, etc.
+const validate: ValidateUpgradeFunc = async (addresses, oldContracts, contracts, logging) => {
+  // display pcvStats
```

Comment:
Being paranoid I would probably make sure that we can move funds here using the newly deployed pcvGuardianV3

Reply:
ok